tests/wsgi_test.py

```diff
@@ -1743,7 +1743,9 @@ def handler(*args):
             fd = self.connect()
             fd.sendall(req.encode())
             fd.close()
-            eventlet.sleep(0.0)
+
+            # TODO changed from 0 to make Python 3 tests pass, not sure if ok
+            eventlet.sleep(0.01)
```

Comment:
From what I saw 2 * ``sleep(0)`` works here just as well; I'll debug it later on and see what events are actually served there.